pkg/plugins/platform/utils.go

```diff
@@ -45,9 +45,13 @@ func GetApps() map[string][]App {
 				}
 
 				err = site.Init(approot)
-				if err == nil {
-					apps[platformType] = append(apps[platformType], site)
+				if err != nil {
+					siteName := siteContainer.Labels["com.ddev.site-name"]
+					appType := siteContainer.Labels["com.ddev.app-type"]
+					site.InitFromMissingDirectory(siteName, appType)
```

Comment:
I don't think it's worth adding the function here. There are only 2 lines in the function, and it probably has a misleading name. So 
* just take those 2 lines and put them here. 
* Add a comment for this stanza saying what the situation is. 
* Also, make sure there aren't other reasons that the site.Init() might fail that would cause misbehavior in this stanza.  (Like do we have a siteContainer OK?)

Reply:
Unfortunately in this case, we can't access the site's AppConfig from this context. This has something to do with the way we are using the App interface, but regardless we need to brainstorm because I agree that it should not have to be its own function...

Reply:
Brainstorm successful: we can work around this situation by casting `site` as type `LocalApp`. From there you can edit values in `.AppConfig`.